js/panel-manager.js

```diff
@@ -0,0 +1,56 @@
+var PanelManager = {
+  TRANSITION_TIME: 300,
+
+  init: function(e) {
+    $('.panel-manager').load('templates/panel-manager.html', function(e) {
+      $('#btnToggle').on('click', function(e) {
+        $('#slider').animate({
+          'right' : $('#slider').css('right') == '0px' ? '-208px' : '0px'
+        }, PanelManager.TRANSITION_TIME);
+        $('#btnToggle').animate({
+          'right' : $('#btnToggle').css('right') == '0px' ? '208px' : '0px'
+        }, PanelManager.TRANSITION_TIME);
+        setTimeout(function(e) {
+          $('#icon-button').toggleClass('glyphicon-chevron-right glyphicon-chevron-left');
+        }, PanelManager.TRANSITION_TIME);
+      });
+    });
+  },
+
+  addPanel: function(viewName) {
+    var viewID = viewName.replace(/\s/g, '');
+    $('#li-init').clone().attr('id', 'li-' + viewID).appendTo('.sideways');
```

Comment:
Please chain the jQuery calls.

Are these ids for the panels? If so please rename them to '#panel-viewID'.